app/models/knight.rb

```diff
@@ -8,5 +8,20 @@ def image
     end
   end
 
+  def valid_move?(new_x, new_y)
+    on_board?(new_x, new_y) && (
+      allowed_deltas = [1, 2]
+      coerced_delta_x = allowed_deltas.delete((x_cord - new_x).abs)
+      coerced_delta_y = allowed_deltas.delete((y_cord - new_y).abs)
+      !!(coerced_delta_x && coerced_delta_y)
+    )
```

Comment:
Okay! I put new method in private.

Thank you both for the guidance on this.  You will probably need to call me out for this issue in the future if I don't improve, please.  Thank you thank you!